frontend/src/App.js

```diff
@@ -161,7 +161,8 @@ function App() {
      * @returns client signature
      */
     function createHardCodedClientSignatureBody() {
-        const regService = new RegisterServiceDto('test', 'https://ipfs.io/ipfs/ID', getIpfsHash());
+        const cid = getIpfsHash();
+        const regService = new RegisterServiceDto('verify_xyz_profiles', 'https://ipfs.io/ipfs/' + cid, cid);
```

Comment:
Set type to "verify_xyz_profiles". For this type, our patched ethr-did-resolve will only check the first (latest)  log-event.